app/src/components/internal/RegistrationFormInternal.svelte

```diff
@@ -17,6 +17,31 @@
 >
   <input class="hidden" type="text" name="fullName" bind:value={$registrationForm.fullName} />
   <input class="hidden" type="text" name="email" bind:value={$registrationForm.email} />
+
+  {#if event.isDigital}
+    <div class="flex flex-col gap-2">
+      <Label for="attendingType">Hvordan vil du delta?</Label>
+      <div class="flex gap-4">
+        <div>
+          <Radio
+            name="attendingType"
+            inline
+            value="Fysisk"
+            bind:group={$registrationForm.attendingType}
+          /> Fysisk
+        </div>
+        <div>
+          <Radio
+            name="attendingType"
+            inline
+            value="Digitalt"
+            bind:group={$registrationForm.attendingType}
+          /> Digitalt
+        </div>
+      </div>
+    </div>
+  {/if}
+
```

Comment:
Denne kunne vært lagt i shared siden den brukes to steder

Reply:
Yes!